### DISCUSSION TITLE
Strange behaviour of pedotransfer function

### DISCUSSION THREAD

**Discussion by diegodinamarca** [mhm-ufz/mHM#73] [Q&A]
Hello mHM team, I am running the model on a basin located in Chile and we are using gridded soil maps with 6 soil horizons (i.e. 0-5, 5-15, 15-30, 30-60, 60-100, 100-200 cm). I used the write_restart = .true. option to check the maps of permanent wilting point (L1_wiltingPoint) and field capacity (L1_soilMoistFC). I divided those values by the thickness of the horizon and made boxplots for each soil layer. As you can see, from the fourth soil layer, FC and PWP values drop to almost cero. In our setup the tillage depth is specified up to 30cm which corresponds to the first three soil layers, is this an expected behaviour of the model? Any help would be very much appreciated.

![FC_mhm](https://github.com/mhm-ufz/mHM/assets/108765210/f446aed9-5247-4c4b-9979-98de7d2a0af7)
![PMP_mhm](https://github.com/mhm-ufz/mHM/assets/108765210/be7f3f50-4a2b-42fc-99cb-574d22a789c7)

**Comment by diegodinamarca** [comment]
To complement my question further. When setting the tillage depth to 2m it gives values for the lower layers but they are the same as all the others.
 
![FC_mhm_tillagedepth2m](https://github.com/mhm-ufz/mHM/assets/108765210/df033a55-0c74-40be-a333-391e298d88ce)
![PMP_mhm_tillagedepth2m](https://github.com/mhm-ufz/mHM/assets/108765210/ce8e6a42-8765-41f2-94d2-afde687a2337)

**Comment by rohinikumar82** [comment]
This is really strange. Can you make the spatial maps of PWP and FC over the study area -- soil layer-wise?

**Comment by diegodinamarca** [comment]
Sure here I show the maps using tillage depth of 2m. These are the raw maps in mm:
Field Capacity 
![FC_mm_6layers](https://github.com/mhm-ufz/mHM/assets/108765210/f86a0f10-8f64-4452-afba-18f13f7386b7)

Permanent Wilting Point
![PWP_mm_6layers](https://github.com/mhm-ufz/mHM/assets/108765210/bf8c80c8-374f-42c5-a29f-e143429e80cf)

And here are the maps after dividing by layer thickness:
Field Capacity
![FC_vol_6layers](https://github.com/mhm-ufz/mHM/assets/108765210/9f38fadc-d369-49d1-84b1-a954a9fa1254)
Permanent Wilting Point
![PWP_vol_6layers](https://github.com/mhm-ufz/mHM/assets/108765210/797ad6da-40e1-4a6f-be19-39be0771ee34)

**Comment by rohinikumar82** [comment]
can u share your mhm.nml file...and also the soil id  and soil look up table files?

**Comment by diegodinamarca** [comment]
Here are the requested files:

[soildata_mhm.zip](https://github.com/mhm-ufz/mHM/files/12731038/soildata_mhm.zip)

**Reply by rohinikumar82** [reply]
thanks.. looking into them.. how did you create these files (look up table)?

**Reply by diegodinamarca** [reply]
the LUT is based on a new soil database created for Chile (https://doi.org/10.1038/s41597-023-02536-x). I extracted the values from all cells to create a table with clay, sand and bulk density; then left only the unique combinations of these three attributes and assigned an ID to each row.

**Comment by rohinikumar82** [comment]
OK - here is my first guess. There is only one soil type (#9) in your setting - over the entire catchment and over all the soil layers; and thatswhy you are getting the same value...
-Rohini

**Comment by diegodinamarca** [comment]
You are absolutely right! we are going to check what happened when creating those files. Thank you for your help. In any case, when the maximum soil depth is lower than the tillage depth limit, is it expected to get the near zero values for FC and PWP at the layers below that limit?

**Reply by rohinikumar82** [reply]
This part, I need to check it more carefully. Sometime later when I start diving into the model code.

**Comment by diegodinamarca** [comment]
Hello, I was talking with a collegue about the results obtained with just one soil type and we wondered why there were differences between the pixels. Does not the pedotransfer function should assign the same values for FC and PMP for all pixels that share the same soil type? Is there another factor that can modify the values of FC and PWP?
